sim/target-agnostic.mk

```diff
@@ -214,13 +214,13 @@ replace-rtl: $(fpga_v) $(ila_work_dir) $(fpga_vh) $(fpga_tcl_env)
 
 .PHONY: replace-rtl
 
-$(firesim_base_dir)/scripts/checkpoints/$(target_sim_tuple): $(fpga_work_dir)/stamp
+$(firesim_base_dir)/scripts/checkpoints/$(name_tuple): $(fpga_work_dir)/stamp
```

Comment:
`target_sim_tuple` isn't defined anywhere either.  It looks like it may have been renamed to `name_tuple`.  I know these targets aren't encouraged to be used but I thought I'd clean them up.

Reply:
Cool, thanks. :)